.circleci/config.yml

```diff
@@ -208,7 +208,7 @@ jobs:
     # triggers a webhook that starts the Jenkins deployment workflow. The convention looks as follows:
     #^(?P<environment>stage|prod)(?:-(?P<task>\w+)-(?P<onfailure>warn|abort))?-(?P<commit>[a-z0-9]+)$
     docker:
-      - image: docker:18.02.0-ce
+      - image: cimg/base:2022.08
         auth:
           username: $DOCKER_USER
           password: $DOCKER_PASS
```

Comment:
I think the DockerHub auth will need to be more explicit? Merino has this: https://github.com/mozilla-services/merino-py/blob/d83ab8ddb4e6669c8bf962286334924e030a75da/.circleci/config.yml#L299